sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/SasClientTests.groovy

```diff
@@ -47,6 +47,41 @@ class SasClientTests extends APISpec {
         sasClient.upload(data.defaultInputStream, data.defaultDataSize)
     }
 
+    @RequiredServiceVersion(clazz = BlobServiceVersion.class, min = "V2020_10_02")
+    def "blob sas all permissions success"() {
+        setup:
+        // FE will reject a permission string it doesn't recognize
```

Comment:
Might be good to read what is actual service version used here and branch out permission creation logic instead of creating service version boundary.
See:
https://github.com/Azure/azure-sdk-for-java/blob/3f31d68eed6fbe11516ca3afe3955c8840a6e974/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/SasClientTests.groovy#L52-L63